### PR TITLE
CAF-1961: Publishing public repos

### PR DESCRIPTION
**Pull request by glucy**
CAF-2034 CAF-2035 CAF-2036 CAF-2037
Update Maven parenting and Distribution Management for Worker-Archetype and Worker-Example.

### CODE REVIEW

File: worker-example/pom.xml [hunk 1]
```diff
@@ -18,7 +18,24 @@
...
+		project will provide a clean well structured starting point for a developer 
+		to create a new worker. -->
+
+	<!-- The worker-framework-aggregator parent is only used here to provide 
+		shared distribution management configuration for the Example Worker. A developer 
+		creating a new project will most likely wish to provide their own distribution 
+		management configuration or remove any distribution management configuration. 
+		Should a developer wish to provide distribution management configuration 
+		for their project they could place the configuration in this aggregator pom.xml 
+		or in each child pom.xml belong to their project. -->
```
Comment (dermot-hardy) [inline worker-example/pom.xml]: These comments don't align with the parent - https://www.youtube.com/watch?v=SsoOG6ZeyUI

**Review by dermot-hardy — COMMENTED** [review]
Should there not be some documentation changes for CAF-2036?

**Comment by glucy** [comment]
Hi @dermot-hardy, the requested changes have now been made:
* Tabs replaced with spaces
* Documentation updated for worker-archetype

File: worker-example/pom.xml [hunk 2]
```diff
@@ -18,7 +18,24 @@
...
+        The worker-archetype project will provide a clean well structured starting 
+        point for a developer to create a new worker. -->
+
+    <!-- The worker-framework-aggregator parent is only used here to provide 
+        shared distribution management configuration for the Example Worker. A developer 
+        creating a new project will most likely wish to provide their own distribution 
+        management configuration or remove any distribution management configuration. 
+        Should a developer wish to provide distribution management configuration 
+        for their project they could place the configuration in this aggregator pom.xml 
+        or in each child pom.xml belong to their project. -->
```
Comment (arogan92) [inline worker-example/pom.xml]: missing "-ing" at the end of belong

File: worker-example/pom.xml [hunk 3]
```diff
@@ -18,7 +18,24 @@
...
+        should a developer wish to create a new worker based upon the Example Worker. -->
+
+    <!-- However should a developer wish to create a new worker similar in 
+        structure to the Example Worker, it is recommended the developer generates 
+        a new project from the provided worker-archetype project under worker-framework. 
+        The worker-archetype project will provide a clean well structured starting 
+        point for a developer to create a new worker. -->
+
+    <!-- The worker-framework-aggregator parent is only used here to provide 
+        shared distribution management configuration for the Example Worker. A developer
```
Comment (arogan92) [inline worker-example/pom.xml:33]: Doesn't quite read right to me. 

"The worker-framework-aggregator parent is only used here to provide **a** shared distribution management configuration for the example worker."

OR

"The worker-framework-aggregator parent is only used here to provide shared distribution management **information/data** for the example worker."

would be my two choices perhaps?

File: worker-example/pom.xml [hunk 4]
```diff
@@ -18,7 +18,24 @@
...
+    <!-- However should a developer wish to create a new worker similar in 
+        structure to the Example Worker, it is recommended the developer generates 
+        a new project from the provided worker-archetype project under worker-framework. 
+        The worker-archetype project will provide a clean well structured starting 
+        point for a developer to create a new worker. -->
+
+    <!-- The worker-framework-aggregator parent is only used here to provide 
+        shared distribution management configuration for the Example Worker. A developer 
+        creating a new project will most likely wish to provide their own distribution 
+        management configuration or remove any distribution management configuration.
```
Comment (arogan92) [inline worker-example/pom.xml:35]: "remove any distribution management configuration"

Same as above. I think that it ending it with the word configuration makes the sentence feel unfinished

File: worker-example/worker-example-shared/src/main/java/com/hpe/caf/worker/example/ExampleWorkerTask.java [hunk 5]
```diff
@@ -32,9 +32,11 @@
     public ReferencedData sourceData;
 
     /**
-     * Identifies a target (relative) location for any output data that worker will save in data store.
-     * If datastore-cs is used, this value will identify a target data store container to use.
-     * If a worker needs to store output data, this should be used when calling {@link com.hpe.caf.api.worker.DataStore#store(Path, String)} method.
+     * Identifies a target (relative) location for any output data that worker will save in data
```
Comment (arogan92) [inline worker-example/worker-example-shared/src/main/java/com/hpe/caf/worker/example/ExampleWorkerTask.java]: "that **the** worker"

**Comment by arogan92** [comment]
Code looks grand, made a couple of comments with the text regarding spelling/grammar. Nothing huge, just nitpicking if I'm honest

File: worker-example/pom.xml (hunk 4 above)
Comment (glucy) [reply worker-example/pom.xml:35]: Leaving as "distribution management configuration"

File: worker-example/pom.xml (hunk 3 above)
Comment (glucy) [reply worker-example/pom.xml:33]: Leaving as "distribution management configuration"

**Comment by glucy** [comment]
@arogan92 Fixed other typos

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**